lib/routes/routeBackbeat.js

```diff
@@ -151,10 +164,59 @@ function putMetadata(request, response, bucketInfo, objMd, log, callback) {
     });
 }
 
+function multipleBackendPutObject(request, response, bucketInfo, objMd, log,
+    callback) {
+    const storageLocation = request.headers['x-scal-storage-class'];
+    const canonicalID = request.headers['x-scal-canonical-id'];
+    if (canonicalID === undefined) {
+        log.error('bad request: missing x-scal-canonical-id header');
+        return callback(errors.BadRequest);
+    }
+    const contentMD5 = request.headers['content-md5'];
+    if (contentMD5 === undefined) {
+        log.error('bad request: missing content-md5 header');
+        return callback(errors.BadRequest);
+    }
+    const sourceVersionId = request.headers['x-scal-version-id'];
+    if (sourceVersionId === undefined) {
+        log.error('bad request: missing x-scal-version-id header');
+        return callback(errors.BadRequest);
+    }
+    const metaHeaders = {
+        'x-amz-meta-scal-location-constraint': storageLocation,
+        'x-amz-meta-scal-replication-status': 'REPLICA',
+        'x-amz-meta-scal-version-id': sourceVersionId,
+    };
+    const context = {
+        bucketName: request.bucketName,
+        owner: canonicalID,
+        namespace: NAMESPACE,
+        objectKey: request.objectKey,
+        metaHeaders,
+    };
+    const payloadLen = parseInt(request.headers['content-length'], 10);
+    const backendInfo = new BackendInfo(storageLocation);
+    return dataStore(context, CIPHER, request, payloadLen, {}, backendInfo, log,
+        (err, retrievalInfo, md5) => {
+            if (err) {
```

Comment:
Please log the error here.

Reply:
I've updated to log errors here, across the routes.